### ISSUE TITLE
Make MaLayout/MaStack/MaColumn compatible with MacOS 11x

### ISSUE THREAD

**Issue by Samitier** [holaluz/margarita#388]
Just a tiny bug I found with browserstack: `gap` doesn't work. We need to use `grid-gap`:

![image](https://user-images.githubusercontent.com/4160121/112995218-4cbe4c00-916b-11eb-8827-2cc0f759030c.png)

Seems like a very easy fix. Is it worth it?

**Comment by afontcu** [comment]
postcss-preset-env should handle it because it is a [stage-3 feature](https://preset-env.cssdb.org/features#gap-properties)

**Comment by afontcu** [comment]
oh, damn – the attribute is added manually on Vue components 🤦  what if… we simply set a CSS custom prop from the component, and let the CSS file handle the styling?

```css
.layout {
  gap: var(--layout-gap);
}
```

then in the component

```js
style() {
  return {
    '--layout-gap': props.gap // or whatever
  }
}
```

I think this would allow postcss to properly transform `gap` (and any other feature, fwiw)

**Comment by afontcu** [comment]
fixed: https://twitter.com/eladsc/status/1387290483433656321

**Comment by Samitier** [comment]
> fixed: https://twitter.com/eladsc/status/1387290483433656321

It's not the same, no? This is `gap` for flexbox (we are using grid). 

Also, the problem is that for MacOS 11x, `gap` is called `grid-gap`. To fix it we should put both `gap` and `grid-gap`. Right now all of our layouts appear without gaps for MacOS 11 or older

**Comment by afontcu** [comment]
You're totally right 😂  (it is not about MacOS but rather Safari 10-11 (released on 2018 with a ~0.1% global usage, though! 😉 ).

I'll take a look anyway, looks like a quick fix.
